.github/workflows/build-with-kokkos.yml

```diff
@@ -88,6 +104,7 @@ jobs:
           cmake --install build-with-${{ matrix.preset }} --prefix=${Kokkos_ROOT}
       - name: Build Kokkos Tools, enabling examples
         run: |
+          export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH};${VARIORUM_ROOT}
```

Comment:
I expect the environment variable already adds it to the search path

Reply:
https://github.com/kokkos/kokkos-tools/blob/dd08b98dafb40f3291704eb54f20cf25cece3b78/cmake/configure_variorum.cmake#L19
Apparently it is spelled with an uppercase `V`.
Please adjust your env variable accordingly and remove the line I highlighted.

Reply:
The previous commit had `Variorum_ROOT` as environemnt variable and didn't find `variorum`. https://github.com/kokkos/kokkos-tools/blob/dd08b98dafb40f3291704eb54f20cf25cece3b78/cmake/configure_variorum.cmake#L8-L10 shows that it should be all uppercase.

Reply:
That configure cmake file logic is absurd and need fixing.
(On my cell can't navigate easily but in short it should not write to Variorum_ROOT that is nonsensical)